src/app/components/Footer.jsx

```diff
@@ -0,0 +1,92 @@
+import SocialMediaButtons from "../components/common/SocialMediaBtn";
```

Comment:
please change the path:
right now you go out of components folder, then go inside the components and then inside common folder
instead:
you just point out to common folder